### PR TITLE
refactor: update home route to match design specs

### PR DESCRIPTION
**Pull request by ismay**
Todo:

- [x] Push down the link in the card. Otherwise the link will move up when there's space.

https://www.sketch.com/s/5f8ebd48-3755-4011-afeb-3e241fc5af9d/a/Dg5dPJ

Route currently looks like this:

Screenshot 2020-09-07 at 16 01 57

### CODE REVIEW

File: src/views/home/Home.module.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    color: var(--colors-grey800);
+    line-height: 20px;
+    font-size: 14px;
+    margin-bottom: var(--spacers-dp24);
+}
+
+.grid {
+    display: flex;
+    flex-wrap: wrap;
+    margin: calc(-1 * var(--spacers-dp8));
```
Comment (cooper-joe) [inline src/views/home/Home.module.css:11]: I think adding `max-width: 800px` makes sense here, the 'card' content becomes difficult to scan/read when stretched across a widescreen monitor.

**Review by cooper-joe — CHANGES_REQUESTED** [review]

File: src/views/home/Home.module.css (hunk 1 above)
Comment (ismay) [reply src/views/home/Home.module.css:11]: Oh yeah, forgot about our container max-width, thanks!

Comment (ismay) [reply src/views/home/Home.module.css:15]: Fixed in 89087aa

**Review by cooper-joe — APPROVED** [review]
LGTM 🚀

**Pull request merged**